core/modules/file/file.field.inc

```diff
@@ -127,7 +127,7 @@ function file_field_instance_settings_form($field, $instance) {
 }
 
 /**
- * Render API callback: Validates the maximum uplodad size field.
+ * Render API callback: Validates the maximum upload size field.
```

Comment:
This doesn't make sense. Perhaps "for the field"?

Reply:
Either that, or *the field's maximum upload size*.

Reply:
It does, since this function is the validator for the 'Maximum upload field':

![image](https://user-images.githubusercontent.com/2385329/220225126-7eeb632d-9b58-4bde-9f1e-a03993ea9169.png)

I'll add quotes to make it more obvious, but I think the wording is correct.

Reply:
In that case, it should be *Render API callback: Validates the "Maximum upload size" field.* It is not a typo, so it is not important for this issue.